invokeai/backend/install/check_root.py

```diff
@@ -7,10 +7,15 @@
 from invokeai.app.services.config import InvokeAIAppConfig
 
 
+# TODO(psyche): Should this also check for things like ESRGAN models, database, etc?
```

Comment:
I mean, can't you have your db and models dirs be in paths totally unrelated to the invokeai root? I think the messaging around this could be changed. Maybe we should just assert on whatever we think about the config would break the app and change the naming accordingly?

Reply:
Renamed